MyApp/app/src/main/java/com/example/janari/myapp/MyWorker.java

```diff
@@ -1,13 +1,18 @@
 package com.example.janari.myapp;
 
+import android.widget.EditText;
+
 /**
  * Created by Janari on 11/20/2017.
  */
 
 public class MyWorker {
```

Comment:
What is `MyWorker`? All names should be descriptive and understandable without extra context..